libcentrifugo/metrics.go

```diff
@@ -109,6 +112,24 @@ type metricsRegistry struct {
 	mu sync.Mutex
 }
 
+func newMetricsHistogramRegistry() *hdrhistogram.HDRHistogramRegistry {
+	quantiles := []float64{50, 90, 99, 99.99}
+	var minValue int64 = 1        // as we record latencies in microseconds, min resolution 1mks
+	var maxValue int64 = 10000000 // as we record latencies in microseconds, max resolution 10s
+	numBuckets := 15              // histograms will be rotated every time we call UpdateSnapshot from outside.
+	sigfigs := 3
+	registry := hdrhistogram.NewHDRHistogramRegistry()
+	registry.Register(hdrhistogram.NewHDRHistogram("http_api", numBuckets, minValue, maxValue, sigfigs, quantiles))
+	registry.Register(hdrhistogram.NewHDRHistogram("client_api", numBuckets, minValue, maxValue, sigfigs, quantiles))
```

Comment:
Without profiling tools it would be anyway hard to make optimizations... I think even more correct way would be measuring this all on Nginx level - because small overhead on Go HTTP library etc... But anyway this is better to have out of the box in Centrifugo than nothing. So I consider this as just a monitoring tool and maybe sth that can show performance improvements or regressions when jumping to new version in production (I think most of our users just make separate `publish` HTTP API requests). 

So yes, let's push it like this then.